.github/workflows/swift-toolchain.yml

```diff
@@ -2131,16 +2239,28 @@ jobs:
         arch: ['amd64' , 'arm64']
 
     steps:
+      - name: Download Debugging Tools
+        uses: actions/download-artifact@v4
+        # TODO(DEVIN-1325)
```

Comment:
This is meaningless, please remove all references to internal issues unless we open up access to Linear. You should move the linear issue to GH and reference that.